src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -489,7 +492,7 @@ void NonClientIslandWindow::_HandleActivateWindow()
     // _titlebarUnscaledContentHeight is set with SetNonClientHeight by the app
     // hosting us.
 
-    _UpdateFrameMargins();
+    THROW_IF_FAILED(_UpdateFrameMargins());

```

Comment:
Was uncertain if throwing was the right thing to do here.

Reply:
Seems like the right thing to me